translator/gobgp.py

```diff
@@ -15,26 +15,32 @@
 DEFAULT_COMMUNITY = 666
 DEFAULT_V4_NEXTHOP = "192.0.2.199"
 DEFAULT_V6_NEXTHOP = "100::1"
+MAX_SMALL_ASN = 2**16
+MAX_SMALL_COMM = 2**16
+IPV6 = 6
 
 logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
 
 
-class GoBGP(object):
+class GoBGP:
     """Represents a GoBGP instance."""
 
     def __init__(self, url):
         """Configure the channel used for communication."""
         channel = grpc.insecure_channel(url)
         self.stub = gobgp_pb2_grpc.GobgpApiStub(channel)
 
-    def _get_family_AFI(self, ip_version):
-        if ip_version == 6:
+    @staticmethod
+    def _get_family_afi(ip_version):
+        if ip_version == IPV6:
```

Comment:
What's the point of variablizing this?

Reply:
I got yelled at for using a "magic number."